include/jemalloc/internal/arena_stats.h

```diff
@@ -50,6 +50,19 @@ struct arena_stats_decay_s {
 	arena_stats_u64_t	purged;
 };
 
+typedef struct arena_stats_extents_s arena_stats_extents_t;
+struct arena_stats_extents_s {
+	/* Total number of extents/bytes in extents_dirty. */
+	atomic_zu_t ndirty;
+	atomic_zu_t dirty_bytes;
```

Comment:
I don't think we need this at all; it can always be derived from ndirty, right? (And similarly, below)

Reply:
Perhaps I'm missing something but my understanding is this: extents are multiples of the page size but not necessary one of the size classes in `SC_NPSIZES`, since the larger size classes have more granular differences than individual pages. While we could get a proper approximation of the number of bytes by just multiplying `ndirty` by the size class size, it isn't exact.

Reply:
Ah, that's a good point, you're right. Could you add a comment explaining?